### ISSUE TITLE
Sync with Caldav tasklist not possible

### ISSUE THREAD

**Issue by cintema** [Armadill0/harbour-tasklist#2]
Unfortunately, your app is missing the feature to sync tasks to OX caldav tasklists.
It would be amazing if you could add this function. SailfishOS allows to add Caldav shares which works fine with calendar - however, task are not displayed. Maybe it is possible to integrate your app with the SailfishOS sync framework? Would be amazing!

**Comment by acesonfire** [comment]
So it will also take a while? :cry: 

Anyway, I could test the caldav support for fruux tasks :wink:

**Comment by Armadill0** [comment]
I hope we'll see some advantages with SailfishOS 2.0 coming with the tablet. But as there's no public roadmap I can't promise anything.

**Comment by Armadill0** [comment]
Unfortunately there's still no annoucement from Jolla when there will be any API for Sailfish services.

I found a point on the Sailfish Dev-Roadmap (https://sailfishos.org/developmentroadmap/):
"Define Sailfish UI extension architecture to allow 3rd party to access the platform’s UI modules and extend the OS through the specified integration points"
But there is no progress on this point.

So I'll move this issue to the 3.0 milestone, as 2.0 is already overdue many months.

**Comment by rednaxela-work** [comment]
Would another option be to include own CalDAV support, eventually using a third party lib?

Of course it is nice to rely on a central system configuration option under account settings and use OS means for syncing but on the other hand, if such a great feature (IMHO) just doesn't come it is really a majore bummer (for me ;-) ).

**Comment by Armadill0** [comment]
Yes, I think it is. The centralized version is the "premium" version, but I don't think it will be possible in the next months.

**Comment by cintema** [comment]
Hey, just came across: https://www.producteev.com/developers

As an alternative, or until caldav-support arrives, you might consider including the Producteev API in the meantime...
Regards,

 Chris

**Comment by Armadill0** [comment]
Some months ago I found a caldav lib implementation: https://cypax.net/projects/iCalendar/

Maybe it can help to integrate CalDAV support anytime soon™. :wink:

**Comment by borghal** [comment]
That would be fantastic!
